### ISSUE TITLE
Freeze after launching the program on Mojave

### ISSUE THREAD

**Issue by BakaPhoenix** [streamlink/streamlink-twitch-gui#758]
## Bug report

- [x] I understand the difference between *Streamlink* and *Streamlink Twitch GUI*.
- [x] This is a bug report and I have read the contribution guidelines.

### Environment details

**Operating system and version:**  
MacOs Mojave 10.14.6

**Streamlink Twitch GUI version:**  
1.10.0

**Streamlink version:**  
1.7.0

### Description
Every time i start the program the mac freeze for several seconds (from 30 sec to a minute)
Same happens when i load my followed streams, this happens only once when i click it the first time, if i do other times after the first i don't have any freeze.

During the freeze i don't get the beachball, i can still move the cursor but everything else is unresponsive

Unistalling the program still have the same problem

### Log output
I made the log using a program called taccy but the log is too long to post it here, despite being only 2 minutes of logs.
https://www.dropbox.com/s/ajm59noivoubm9c/twitch%20gui.rtf?dl=0
### Additional comments, screenshots, etc.

I think is something related to tccd process? right before the freeze and after i see this process writing around 40mb/s on my main hd but i can't figure out what can be causing this problem

**Comment by bastimeyer** [comment]
Duplicate of #688.

Post a **proper debug log** there, and only use **plain text** in a [Github gist](https://gist.github.com/). If there's actually useful information, we can reopen that thread again. Other than that, try to disable GPU acceleration.
https://github.com/streamlink/streamlink-twitch-gui/wiki/Parameters#nwjs--chromium-parameters

I will see if I can upgrade the used NW.js version in the next release (there have been a couple of issues previously a few months ago), but that's all I can do here.
